src/Support/Collection/MessageCollection.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace OwowAgency\Gossip\Support\Collection;
+
+use Illuminate\Database\Eloquent\Collection;
+use OwowAgency\Gossip\Models\Contracts\HasConversationContract;
+
+class MessageCollection extends Collection
+{
+    /**
+     * Mark messages as read if the force parameter is set to true.
+     *
+     * @param  \OwowAgency\Gossip\Models\Contracts\HasConversationContract  $model
+     * @param  bool  $force
+     * @return \OwowAgency\Gossip\Support\Collection\MessageCollection
+     */
+    public function markAsRead(HasConversationContract $model, bool $force = false): MessageCollection
+    {
+        if ($force) {
```

Comment:
Looks like an implementation that should not be present in this method. 

Only drunk people will check after penetration if they are wearing a condom.

Reply:
I was drunk and in love when writing this..